Event/include/Event/TrigScintCluster.h

```diff
@@ -0,0 +1,191 @@
+/**
+ * @file TrigScintCluster.h
+ * @brief Class that stores cluster information from the TrigScint 
+ * @author Josh Hiltbrand, University of Minnesota
+ * @author Lene Kristian Bryngemark, Stanford University : adaptation to TrigScint 
+ */
+
+#ifndef EVENT_TRIGSCINTCLUSTER_H_
+#define EVENT_TRIGSCINTCLUSTER_H_
+
+// ROOT
+#include "TObject.h"
+#include "TString.h"
+#include "TRefArray.h"
+
+// STL
+#include <iostream>
+#include <set>
+
+// ldmx-sw
+#include "Event/EventConstants.h"
+#include "Event/TrigScintHit.h"
+#include "Event/EcalCluster.h"
+// these two can't be found..?
+//#include "Framework/ParameterSet.h" // Needed to import parameters from configuration file
+//#include "Framework/EventProcessor.h"
```

Comment:
You can delete these. The Event module _should not_ depend on the Framework module, so if it compiles without these, you can skip them.